src/index.css

```diff
@@ -0,0 +1,238 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,500;0,600;0,700;1,400&display=swap');
+
```

Comment:
- [ ] It would be nice if you stick to the design guideline. It's required that you use Lato font for the body font of the project as described on Behance. We're trained to follow design guidelines as this is exactly how we'll have to follow instructions in the real job field. Kindly work on that 👍🏽

<img width="1400" alt="Screenshot 2023-04-28 at 11 59 37 AM" src="https://user-images.githubusercontent.com/50941074/235148101-88870401-5135-423b-baae-9214404a753b.png">

Reply:
@AmaduKamara 
Thank you for reviewing.
All the required changes are fixed [here](https://github.com/ZewdieMc/covid-19-cases/pull/14/commits/9715e72cf4e165daeae26614903fde4d86c817fa).